src/VisualStudio/CSharp/Impl/Options/AutomationObject.cs

```diff
@@ -275,6 +275,12 @@ public int NewLines_Braces_ObjectInitializer
             set { SetBooleanOption(CSharpFormattingOptions2.NewLinesForBracesInObjectCollectionArrayInitializers, value); }
         }
 
+        public int NewLines_Braces_WithInitializer
+        {
+            get { return GetBooleanOption(CSharpFormattingOptions2.NewLinesForBracesInWithInitializer); }
+            set { SetBooleanOption(CSharpFormattingOptions2.NewLinesForBracesInWithInitializer, value); }
+        }
```

Comment:
I added this following the implementation of other `NewLines_*` options. But I don't see any of these properties used anywhere. Are these used by an internal implementation that's not open-source? Will that implementation need to be updated to benefit from this new property in some way?